.github/workflows/auto-maintenance.yml

```diff
@@ -1,4 +1,4 @@
-name: Coverage pages
+name: Automatic maintenance
```

Comment:
```suggestion
name: Automatic Maintenance
```

I'd recommend title case for the name

Reply:
Why would you capitalize that?

Reply:
To comply with [title case](https://en.wikipedia.org/wiki/Title_case). It's good fashion to use it in scenarios like this